std_msgs/Header header
std_msgs/Int32 id
geometry_msgs/PoseStamped observerPose
geometry_msgs/PoseStamped HomePose 
geometry_msgs/PoseStamped robotReleasePose

std_msgs/Float32 target_height

std_msgs/Bool dobot_action_type
std_msgs/Bool dobot_action1
std_msgs/Bool dobot_action2
std_msgs/Bool dobot_action3
std_msgs/Bool dobot_action4
std_msgs/Bool dobot_action5
